clients/privacy-center/config/config.css

```diff
@@ -1 +1,19 @@
-/* Add global CSS overrides to this file */
+/*
+Add any global CSS overrides to this file.
+Override basic theme colors by uncommenting and editing those below
+*/
+
+:root:root {
+    /* Background color */
+    /* --chakra-colors-gray-50: #F7FAFC; */
+    /* Header & highlight color */
+    /* --chakra-colors-gray-100: #EDF2F7; */
+    /* Modal text color */
+    /* --chakra-colors-gray-500: #718096; */
+    /* Body text color */
+    /* --chakra-colors-gray-600: #4A5568; */
+    /* Primary button hover color */
+    /* --chakra-colors-primary-400: #464B83; */
+    /* Primary button color */
+    /* --chakra-colors-primary-800: #111439; */
```

Comment:
I went through the exercise of theme-ing a privacy center and this was really painful to hunt down these theme colours... this still isn't ideal, but it provides some hints and living documentation for users in the future...